<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>windowOfLogin</class>
 <widget class="QWidget" name="windowOfLogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
     <item>
      <widget class="QWidget" name="boxOfLogin" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QGridLayout" name="gridLayout_2" rowminimumheight="0,0">
        <item row="1" column="0">
         <widget class="QLabel" name="textOfPassword">
          <property name="text">
           <string>密码:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="textOfAccount">
          <property name="text">
           <string>账号:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLineEdit" name="inputOfAccount">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLineEdit" name="inputOfPassword">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="buttonsOfLogin" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QPushButton" name="buttonOfReset">
          <property name="text">
           <string>复位</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="buttonOfLogin">
          <property name="text">
           <string>登录</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="action">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="importAction">
   <property name="text">
    <string>导入</string>
   </property>
  </action>
  <action name="exportAction">
   <property name="text">
    <string>导出</string>
   </property>
  </action>
  <action name="clearAction">
   <property name="text">
    <string>清除</string>
   </property>
  </action>
  <action name="exitAction">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="resetAction">
   <property name="text">
    <string>复位</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
